app/src/test/java/com/codesoom/assignment/domain/TaskTest.java

```diff
@@ -0,0 +1,20 @@
+package com.codesoom.assignment.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskTest {
+
+    @DisplayName("task 객체를 생성할 수 있다")
```

Comment:
테스트의 목표, 목적을 정확히 기술하는 것이 중요합니다. 이 테스트는 단순하기 때문에 의외로 놓치기 쉬운 것 같아요.
테스트 코드를 잘 읽어보면 task 객체를 생성하는 내용의 테스트가 아니라는 것을 알 수 있습니다. 이 테스트는 `setter`와 `getter`를 테스트하고 있습니다.

다음은 제가 작성한 테스트 설명문인데요, 테스트 내용과 비교해서 읽어보시고 테스트 본문이나 설명문을 수정해 보세요.
```suggestion
    @DisplayName("task 객체를 생성하고 setter로 값을 설정하면 getter로 설정한 값을 리턴받을 수 있다 ")
```

그리고 나머지 테스트에 대해서도 살펴보세요.